ledger/src/blockstore_db.rs

```diff
@@ -169,7 +169,13 @@ pub mod columns {
     pub struct ErasureMeta;
 
     #[derive(Debug)]
-    /// The bank hash column
+    /// The bank hash column.
+    ///
+    /// This column family persists the bank hash of recent each slot.
```

Comment:
"of recent each slot" ==> "for a given slot"

We technically may not have a hash for each slot (ie dead slot)